package/lib/index.js

```diff
@@ -34,7 +34,7 @@
  */
 function suggest (incorrectName, correctName) {
```

Comment:
Please add some mark / hint that this method should be removed after migration has been completed and the package has been in use for a while.

Reply:
I hadn't intended for it to be temporary.  I believe it is helpful, for example, for when a person types `A.isStrictSubset(B)` instead of `A.isProperSubset(B)`.  The terminology "strict subset" is established and unambiguous, and it is synonymous with "proper subset".